### ISSUE TITLE
Option to do only group sorting, not alphabetical sort

### ISSUE THREAD

**Issue by hydrargyrum** [PyCQA/isort#1191]
There is this option

```
  -dt, --dont-order-by-type
                        Only order imports alphabetically, do not attempt type ordering
```

but it would be nice to have the opposite: sort by group (stdlib, application, etc.) and nothing else

**Comment by anirudnits** [comment]
Hi! if no one is working on this, I would like to work on this issue.

**Comment by timothycrosley** [comment]
@anirudnits I'm sorry I'm so late to seeing this! If you still would like to work on it, it's now assigned to you.

**Comment by anirudnits** [comment]
@timothycrosley no worries :) I am facing some problems with getting the setup done in my WSL(it was fine in pure Ubuntu) and trying to do it via Docker. Will start working on the issue as soon as that's done.

**Comment by anirudnits** [comment]
@timothycrosley Hi, I just had a few questions and would appreciate it if you could clear them out.
-  should I add a new CLI flag? like `--only-sections` or something like that and another attribute in the config data schema
- would the order of `straight_imports` over `from_imports` hold if not specified otherwise?

**Comment by timothycrosley** [comment]
Hi @anirudnits,

Happy to answer any questions!

> should I add a new CLI flag? like --only-sections or something like that and another attribute in the config data schema
Yes, exactly!

> would the order of straight_imports over from_imports hold if not specified otherwise?
Yes I think it makes sense for it to hold if not otherwise specified

Thanks!

~Timothy

**Comment by anirudnits** [comment]
@timothycrosley Sorry to bother you again. I'm sort of done with the code but when I try to test with `docker build ./ -t isort:latest`. I get this error:
`
/isort/isort does not contain any element
The command '/bin/sh -c python -m pip install --upgrade pip && python -m pip install poetry && poetry install' returned a non-zero code: 1
`
I did the docker setup earlier and I guess this showed up after I ran the `git pull upstream develop` command. Is this a reproducible error or am I doing something wrong?
